src/index.scss

```
@@ -244,3 +176,4 @@ main {
         width: 1024px;
     }
 }
+@import "./themes/magic.scss";
```

Stylelint (17.11.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)